prime-angular-frontend/src/app/modules/sat-eforms/shared/modules/sat-eforms-login-page/sat-eforms-login-page.component.html

```diff
@@ -2,4 +2,26 @@
 
 <app-simple-access [title]="title"
                    [loginLabel]="loginLabel"
-                   (login)="onLogin()"></app-simple-access>
+                   (login)="onLogin()">
+
+  <div class="row">
+    <div class="col col-lg-6 pb-3">
+      <p>
+        Healthcare professionals in BC can enrol with this application to get access to the PharmaCare Special
+        Authority eForms application. After you enrol here, you will be able to log into PharmaCare Special
+        Authority eForms with your BC Services Card.
+      </p>
+      <p>
+        If you do not have your mobile BC Services Card set up, please visit the BC Services Card website for
+        instructions.
```

Comment:
Not a functional issue but was thinking for convenience, should we provide a hyperlink to the BC Services Card website?

Reply:
Yes, I had to wait for Sarah to provide us with one that wasn't broken.